GeneralsMD/Code/GameEngineDevice/Include/W3DDevice/GameClient/W3DGameClient.h

```diff
@@ -111,8 +114,11 @@ class W3DGameClient : public GameClient
 
   /// Manager for display strings
 	virtual DisplayStringManager *createDisplayStringManager( void ) { return NEW W3DDisplayStringManager; }
-
+#ifdef RTS_HAS_FFMPEG
```

Comment:
There is `RTS_USE_OPENAL` in the other file. I think "USE" reads better than "HAS". Perhaps stick to "USE" for these things?

Reason being, "having" something is different to "using" something. So for example we could have ffmpeg ready to use, but still use bink, right?

Therefore I think "USE" or "USING" is better.